lib/position.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Position
+  $count = 0
+  attr_accessor :get_position
```

Comment:
Use `attr_reader` instead of `attr_accessor` is you are only going to read, not write, the variable. Check your use of `attr_accessor`.